users/views.py

```diff
@@ -14,3 +21,18 @@ def post(self, request):
             content = {'message': 'wrong google token / this google token is already expired.'}
             return Response(content)
         return Response(r)
```

Comment:
```suggestion:-1+1
        return Response(r)
```
Also remove this line.